cloud-service-providers/google-cloud/vertexai/python/airgap/docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+services:
+  nim-llama-8b:
+    image: nvcr.io/nim/meta/llama3-8b-instruct:latest
+    shm_size: 35gb
```

Comment:
Should this be 35? I thought the official docs recommended smaller.

Reply:
Yes, official recommended shm_size is 16GB, it's used for original NIM, size ~12.5GB. For the Airgap NIM the new image size grows to ~29GB, so increase to 35GB here to avoid shared memory creation error. Do we have alternative solution recommended?

Reply:
@supertetelman Any suggestions here?

Reply:
I don't understand, I will need to take a closer look at this code. I did not think we were building any custom Docker image or modifying the startup command of the Docker image in  a way that would necessitate this change.

Would you mind pointing out the line of code where you think this becomes necessary?

Reply:
In this docker-compose.yaml line 17-23 bind volumes may cause the increasing size of the new docker image.

Reply:
This is the only pending item we have now. Please let me now your thoughts and suggestions. Thank you!